### PR TITLE
[tools]Update pack.py to support cordova4.0 packing.

### PR DESCRIPTION
**Pull request by jiajiax**

### CODE REVIEW

**Comment by jiajiax** [comment]
@Honry Please update suite.json and verify this patch.

**Comment by Honry** [comment]
Please add the UT result.

File: tools/build/pack.py [hunk 1]
```diff
@@ -623,14 +623,17 @@ def packCordova(build_json=None, app_src=None, app_dest=None, app_name=None):
         os.chdir(orig_dir)
         return False
     os.chdir(os.path.join(BUILD_ROOT, "cordova", app_name))
-    pack_cmd = "./cordova/build"
+    if BUILD_PARAMETERS.subversion:
```
Comment (Honry) [inline tools/build/pack.py]: What if subversion=3.6? Please consider this situation.

Comment (jiajiax) [inline tools/build/pack.py]: OK , this patch need to be updated , I will do it ASAP.

**Comment by jiajiax** [comment]
I have checked that cordova4.0 and cordova3.6 suites can be packed normally with this patch.

**Comment by Honry** [comment]
LGTM @jennycao, please take a review. :smile:

**Pull request merged**